server/tests/unit/address.test.js

```diff
@@ -0,0 +1,138 @@
+const router = require("../../routes/order/address");
+const request = require("supertest");
+
+const id;
+
+describe("address form validation", () => {
+  test("address form should return 400 for empty body", async () => {
+    request(router).post("/").send({}).expect(400);
+  });
+
+  test("user Validation should return 400 for invalid email", async () => {
+    request(router)
+      .post("/")
+      .send({
+        name: "gangotri",
+        email: "notanemail",
+        phone: 9845521479,
+        street: "kote road",
+        city: "Rattihalli",
+        pin_code: 581116,
+        state: "karnataka",
+        country: "India",
+      })
+      .expect(400);
+  });
+  test("address form should return 400 for invalid name", async () => {
+    request(router)
+      .post("/")
+      .send({
+        name: "",
+        email: "nadigergangotri@gmail.com",
+        phone: 9845521479,
+        street: "kote road",
+        city: "Rattihalli",
+        pin_code: 581116,
+        state: "karnataka",
+        country: "India",
```

Comment:
can you use faker npm package to generate random fake data instead of using your data?

Reply:
Data is repeated so many times!!